### PR TITLE
Translation to Spanish

### PR DESCRIPTION
**Pull request by Include-Poetry**

### CODE REVIEW

File: client/app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">MrHyde</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-es/strings.xml]: Please remove this line, we won't translate that in the future :)

File: client/app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    <string name="markdown_h3">Título H3</string>
+    <string name="markdown_h4">Título H4</string>
+    <string name="markdown_h5">Título H5</string>
+    <string name="markdown_h6">Título H6</string>
+    <string name="markdown_ul">Lista de la selección</string>
+    <string name="markdown_ol">Lista ordenada de la selección</string>
+    <string name="markdown_quote">Cita de la selección</string>
+    <string name="markdown_code">Bloque de código de la selección</string>
+
+    <string name="markdown_element_bold">**</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-es/strings.xml]: Please remove the `markdown_element_*` entries. Those are markdown formatting elements that don't need translation.

File: client/app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+    <string name="action_undo">Deshacer</string>
+    <string name="action_redo">Rehacer</string>
+    <string name="action_publish_draft">Publicar borrador</string>
+    <string name="action_unpublish_post">Convertir a borrador</string>
+    <string name="action_insert_markdown">Insertar Markdown</string>
+    <string name="action_open_in_browser">Abrir en el navegador</string>
+    <string name="action_help">Ayuda</string>
+
+    <string name="select_git_hoster_title">Seleccionar proveedor</string>
+    <string name="github">GitHub</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-es/strings.xml]: Please remove this and the next two entries. No translation needed :)

File: client/app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    <string name="drafts">Borradores</string>
+    <string name="draft">Borrador</string>
+    <string name="new_draft">Nuevo borrador</string>
+    <string name="no_drafts">Sin borradores todavía</string>
+    <string name="all_files">Todos los archivos</string>
+    <string name="view_all_files">Ver todos los archivos</string>
+
+    <string name="filename">Nombre del archivo</string>
+    <string name="your_awesome_title">Tu asombroso título</string>
+    <string name="default_front_matter">---\nlayout: post\ntitle: %1$s\n---\n</string>
```
Comment (Maddoc42) [inline client/app/src/main/res/values-es/strings.xml]: Please remove the this. The default front matter does not need to be translated.

**Comment by Include-Poetry** [comment]
Done

**Comment by Maddoc42** [comment]
Looks great! :+1: thanks again for your help, much appreciated :)

Would you mind being mentioned in the credits / about section?

**Comment by Include-Poetry** [comment]
Happy to help. That would be great 👍

**Pull request merged**